tutorials/commands/flux-job-cancel.rst

```diff
@@ -0,0 +1,114 @@
+.. _flux-job-cancel:
+.. _flux-job-cancelall:
+
+========================
+How to Cancel a Flux Job
+========================
+
+Inevitably submitted jobs will have to be canceled for one reason or another.  This tutorial
+will show you how.
+
+----------------------
+Basic Job Cancellation
+----------------------
+
+The basic way to cancel a job is through ``flux job cancel``.  All you have to do is specify
+the jobid on the command line.  Here is a simple example after submitting a job.
+
+.. code-block:: console
+
+    $ flux mini submit sleep 100
+    ƒh35Dh5qRyq
+
+    $ flux jobs ƒh35Dh5qRyq
+           JOBID USER     NAME       ST NTASKS NNODES     TIME INFO
+     ƒh35Dh5qRyq achu     sleep       R      1      1   13.33s corona174
+
+    $ flux job cancel ƒh35Dh5qRyq
+
+    <snip wait a little bit>
+
+    $ flux jobs ƒh35Dh5qRyq
+           JOBID USER     NAME       ST NTASKS NNODES     TIME INFO
+     ƒh35Dh5qRyq achu     sleep      CA      1      1   20.18s corona174
+
+In the above example we submitted a simple job via ``flux mini submit`` that simply
+runs ``sleep``.  Passing the resulting jobid to ``flux jobs`` shows that it is
+running (state is ``R``).
+
+We cancel the job simply by passing the jobid to ``flux job cancel``.  After waiting
+a little bit, we see that the job is now canceled in ``flux jobs`` (state is ``CA``).
+
+------------------------
+Cancelling a lot of jobs
+------------------------
+
+The special ``flux job cancelall`` command allows you to cancel many jobs.  Several options allow you to
+target a specific set of jobs.
+
+To start off, lets create 100 jobs that will sleep infinitely.  We will use the special ``--cc`` option
```

Comment:
oh no, sleeping beauty jobs!

Reply:
What does `--cc` mean?

Reply:
i assume "carbon copy", which we can mention (and hopefully its obvious what it does given what its called)